auacm/angular/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AngularPage } from './app.po';
+
+describe('angular App', function() {
+  let page: AngularPage;
+
+  beforeEach(() => {
+    page = new AngularPage();
+  });
+
+  it('should display message saying app works', () => {
```

Comment:
Yes, all spec.ts fils are test files. They are generated by Angular-cli and I left them in here if we decide to do front-end testing. I can remove them though if you want.